SpringProject/src/main/java/com/gradproject2019/users/repository/UserRepository.java

```diff
@@ -30,12 +30,23 @@ void updateUser(@Param("id") Long id,
                     @Param("email") String email,
                     @Param("occupation") String occupation);
 
-    @Query(value = "SELECT * FROM users WHERE username = :username", nativeQuery = true)
+    @Query(value = "SELECT * " +
+            "FROM users " +
+            "WHERE username = :username",
+            nativeQuery = true)
     Optional<User> findByUsername(@Param("username") String username);
 
-    @Query(value = "SELECT * FROM users WHERE email = :email", nativeQuery = true)
+    @Query(value = "SELECT * " +
+            "FROM users " +
+            "WHERE email = :email",
+            nativeQuery = true)
     Optional<User> findByEmail(@Param("email") String email);
 
-    @Query(value = "SELECT * FROM users WHERE first_name like CONCAT('%', :query,'%') OR last_name like CONCAT('%', :query, '%') ORDER BY last_name ASC", nativeQuery = true)
+    @Query(value = "SELECT * " +
+            "FROM users " +
+            "WHERE CONCAT(first_name, ' ', last_name)" +
```

Comment:
make sure to leave spaces at the start of each line - this will all form one long string